torchao/quantization/quantize_/workflows/float8/float8_tensor.py

```diff
@@ -256,6 +303,9 @@ def _(func, types, args, kwargs):
                 kernel_choice = "fbgemm"
         elif weight_tensor.kernel_preference == KernelPreference.FBGEMM:
             kernel_choice = "fbgemm"
+        elif weight_tensor.kernel_preference == KernelPreference.TRITON:
+            # no triton gemm op is available, so we'll fallback to torch
+            kernel_choice = "torch"
```

Comment:
also weird, your `kernel choice` is doing double duty and now a recipe. that recipe is also not very clear from your initial doc

Reply:
do you mean kernel choice is used for both quantize and gemm?

what do you mean by initial doc?

Reply:
> kernel choice is used for both quantize and gemm

yeah that's a decision we made before, according to Josh there is no need to have kernel level choice for now, just to keep things simple.

Reply:
IMO it should be:
* fbgemm - use all fbgemm kernels, error out if something is not supported
* torch - use all torch kernels, error out if something is not supported
* auto - torchao decides what to do

we should not use torch kernels in the fbgemm setting, as that is not honoring what the user asked for